docker-entrypoint.sh

```diff
@@ -365,6 +395,9 @@ docker_setup_db() {
 		fi
 	fi
 
+	# To create replica user
+	create_replica_user
```

Comment:
something here like:

`local createReplicaUser = $(create _replica_user)`

Include `${createReplicaUser` beside the `${createUser}` below.

Reply:
Done 👍🏽